tests/regression/80-termination/09-complex-for-loop-terminating.c

```diff
@@ -0,0 +1,107 @@
+// SKIP LOCAL_TERM PARAM: --set "ana.activated[+]" termination --enable warn.debug --set ana.activated[+] apron --enable ana.int.interval --set ana.apron.domain polyhedra
```

Comment:
Is there a reason this test is skipped? Is it because Goblint cannot determine the precise verdict? In that case, it may be helpful to introduce an annotation for tests that indicates that termination is the proper verdict but that Goblint is currently unable to determine that.